master/db.ffnord

```diff
@@ -29,6 +29,10 @@ gw02  A     10.187.1.2
 1.ntp.services  AAAA  2a03:2267::101	;ffhh srv01
 2.ntp.services  AAAA	2a03:2267::102	;ffhh srv02
 
+; Knotenforumlar
+formular.ffnord	CNAME	srv01
+formular		CNAME	formular.ffnord
```

Comment:
why these two CNAMEs?

this would translate to

    formular.ffnord.ffnord. CNAME srv01.ffnord.
    formular.ffnord. CNAME formular.ffnord.ffnord.

is that what you intended to do?